src/pages/ResearcherProfile.js

```diff
@@ -162,6 +126,52 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
     }
   };
 
+  handleChangeDisplayName = (e) => {
+    const value = e.target.value;
+    this.setState(prev => {
+      prev.currentUser.displayName = value;
+      return prev;
+    });
+  };
+
+  handleAdditionalEmailChange = (e) => {
```

Comment:
handleAdditionalEmailChange, handleCheckboxChange, are not new they have just been moved to keep all the handle methods in one place